libs/components/lists/testing/src/modules/list-summary/list-summary-harness.spec.ts

```diff
@@ -0,0 +1,189 @@
+      <sky-list-summary-item
+        data-sky-id="test-item-with-help"
+        [labelText]="'Revenue'"
+        [value]="1000000"
+        [helpPopoverContent]="
+          'This represents the total revenue for the current period.'
+        "
+        [helpPopoverTitle]="'Revenue Help'"
+      />
+      <sky-list-summary-item
+        data-sky-id="test-item-without-help"
+        [labelText]="'Profit'"
+        [value]="250000"
+      />
+    </sky-list-summary>
+  `,
+})
+class TestComponentWithHelp {}
+
+let fixture: ComponentFixture<TestComponent | TestComponentWithHelp>;
+let loader: HarnessLoader;
+
+async function setupTest(
+  component: typeof TestComponent | typeof TestComponentWithHelp,
+): Promise<void> {
+  await TestBed.configureTestingModule({
+    imports: [component],
+    providers: [provideNoopAnimations()],
+  }).compileComponents();
+
+  fixture = TestBed.createComponent(component);
+  loader = TestbedHarnessEnvironment.loader(fixture);
+}
+
+describe('List Summary Harnesses', () => {
+  describe('SkyListSummaryHarness', () => {
+    beforeEach(async () => {
+      await setupTest(TestComponent);
+    });
+
+    it('should locate the summary component', async () => {
+      const harness = await loader.getHarness(SkyListSummaryHarness);
+      expect(harness).toBeTruthy();
+    });
+
+    it('should locate the summary component by dataSkyId', async () => {
+      const harness = await loader.getHarness(
+        SkyListSummaryHarness.with({ dataSkyId: 'test-summary' }),
+      );
+      expect(harness).toBeTruthy();
+    });
+
+    it('should get all summary items', async () => {
+      const harness = await loader.getHarness(SkyListSummaryHarness);
+      const items = await harness.getSummaryItems();
+      expect(items.length).toBe(2);
+    });
+
+    it('should get a specific summary item', async () => {
+      const harness = await loader.getHarness(SkyListSummaryHarness);
+      const item = await harness.getSummaryItem({ labelText: 'Total Items' });
+      expect(item).toBeTruthy();
+    });
+  });
+
+  describe('SkyListSummaryItemHarness', () => {
+    beforeEach(async () => {
+      await setupTest(TestComponent);
+    });
+
+    it('should get the label text', async () => {
+      const harness = await loader.getHarness(
+        SkyListSummaryItemHarness.with({ dataSkyId: 'test-item-1' }),
+      );
+      const labelText = await harness.getLabelText();
+      expect(labelText).toBe('Total Items');
+    });
+
+    it('should get the value text', async () => {
+      const harness = await loader.getHarness(
+        SkyListSummaryItemHarness.with({ dataSkyId: 'test-item-1' }),
+      );
+      const valueText = await harness.getValueText();
+      expect(valueText).toBe('42');
+    });
+
+    it('should filter by label text', async () => {
+      const harness = await loader.getHarness(
+        SkyListSummaryItemHarness.with({ labelText: 'Average Score' }),
```

Comment:
```suggestion
        SkyListSummaryItemHarness.with({ labelText: 'Average score' }),
```
I assume that if we make the other changes, we should change the capitalization here too.